### ISSUE TITLE
Redesign website

### ISSUE THREAD

**Issue by ericmj** [hexpm/hex_web#25]
We want to move away from the temporary bootstrap design. @scrogson has already shown interest in helping out here. If you'd like to help or have some input please post in this issue.

**Comment by milmazz** [comment]
@ericmj @jcspencer How are you tackling this issue? If you want I can help with the _Statistics_ area at the beginning, I have previous experience with [Highcharts](http://www.highcharts.com/) but I can try [Chart.js](http://www.chartjs.org/).

**Comment by ericmj** [comment]
@milmazz No one is working on this issue. Please contribute anything you can! :+1:

**Comment by ericmj** [comment]
Closing this as the issue is too general. Will open an issue for stats.

**Comment by tomekowal** [comment]
Can we also add something like "most popular searches" feature? I know, it is usually not super important, but having some kind of statistics could help developers decide what other libraries people need.

**Comment by ericmj** [comment]
Yes, it could be useful but before we start adding features we need someone that will do the actual work. I don't have the time or the design knowledge to do it myself.

**Comment by pragmaticivan** [comment]
Is there plans for this Task? I started something, but I'm not sure how this project is managed.

![preview](https://cloud.githubusercontent.com/assets/301291/11321972/7e53328c-9089-11e5-9a98-2f80bfa6716b.png)

**Comment by pragmaticivan** [comment]
Is there plans for this Task? I started something, but I'm not sure how this project is managed.

![preview](https://cloud.githubusercontent.com/assets/301291/11322042/a0981d6e-908c-11e5-9ca8-e41f58d4b6a9.png)

**Comment by ericmj** [comment]
There are definitely still plans for this. This is something I don't have skills for myself so I left the issue open as a way for better designers (like you) to find it :).

I like what you have so far, it looks great. But I am not convinced about the purple background, especially since it won't fit with the new logo that we have not released yet. I can send you a draft of the logo privately if you would like.

**Comment by pragmaticivan** [comment]
Cool, please send me this. I just created the colors based on Elixir. Because the current website doesn't have a lot of pallet context. But It's just a prototype. I can change anything.

**Comment by pragmaticivan** [comment]
I'm not sure if I can post the draft of the logo here, but let me know and I can delete.

![previewhex](https://cloud.githubusercontent.com/assets/301291/11322805/d13109ac-90b1-11e5-8499-2705b7deb212.png)

**Comment by josevalim** [comment]
Wow, this looks really nice! Two tiny suggestions:

For the main text, I would suggest the following:

```
The package manager for the Erlang ecosystem
    [Find packages                    ]
```

Also, we don't typically show who published the package, so you should consider this information won't be shown or won't be available as well. :)

**Comment by pragmaticivan** [comment]
So if we have this information about the package, should we show the info about who published the package?
<img width="281" alt="screen shot 2015-11-22 at 1 03 23 am" src="https://cloud.githubusercontent.com/assets/301291/11322888/f39a21ce-90b4-11e5-835e-f9312f4ed9cc.png">

**Comment by pragmaticivan** [comment]
Fixed.

**Comment by josevalim** [comment]
Yes, we should never show who published the package. :) We focus on the maintainers, the particular person who published is not that important. :)

**Comment by pragmaticivan** [comment]
Cool, thanks.

**Comment by pragmaticivan** [comment]
Updated with all the changes.
![previewhex](https://cloud.githubusercontent.com/assets/301291/11322931/29daae10-90b6-11e5-91a1-2f4637da9488.png)

**Comment by victorlcampos** [comment]
+1

**Comment by henrik** [comment]
@ericmj:
> So tags may be a good idea, I need to think some more about that 

You can see some tagging in action at http://toolbox.elixir.pm (site is still rather rough), which basically mirrors the Hex.pm package list and adds tags/multiple categories on top. I'm not sure myself yet if it's a good way of organizing/discovering packages – it's an experiment (and an excuse to learn Phoenix).

If I find the time or inclination (or someone contributes it), I'm happy to try out things like user pictures, READMEs etc there, and if it seems valuable maybe Hex.pm could adopt some of those things.

**Comment by ericmj** [comment]
I think it's an excellent idea to try out new features separately and then discuss if they should be added to Hex itself. Great initiative @henrik!

I also want to clarify that the discussion in the early posts is pretty old and and some of the ideas have already been implemented, abandoned or put on the back burner. I'm not dismissing any of the above ideas, some of them may be nice to have, but none of them are high priority for the Hex team at this time. If you would like to work on one of them please open an issue and we can discuss it separately.

@pragmaticivan feel free to open a new issue with your design and I will close this issue as the initial, old posts are more confusing than helpful at this point.

**Comment by ericmj** [comment]
Closed in favor of #188.
